src/components/card/card.tsx

```diff
@@ -0,0 +1,250 @@
+  className?: string;
+  /** Whether the card is clickable. */
+  isClickable?: boolean;
+  /** Called when the card is clicked. */
+  onClick?: () => void;
+  /** Optional actions to display in the top right corner. */
+  actions?: CardAction[];
+  /** Whether actions should only be visible on hover (default: true - actions always visible). */
+  showActionsOnHover?: boolean;
+  /** Whether to group actions into a dropdown menu (default: false - show as individual icon buttons). */
+  groupActions?: boolean;
+}
+
+/*
+ * Style.
+ */
+
+interface StyledCardProps {
+  $size: VisualSizesEnum;
+  $hasShadow: boolean;
+  $hasBorder: boolean;
+  $isClickable: boolean;
+}
+
+const cardPadding = makeSizeConstants(12, 16, 20);
+const cardBorderRadius = makeSizeConstants(6, 8, 10);
+
+const StyledCard = styled.div<StyledCardProps>`
+  display: flex;
+  flex-direction: column;
+  background: ${greys.white};
+  border-radius: ${(p) => cardBorderRadius[p.$size]}px;
+  box-sizing: border-box;
+  overflow: hidden;
+  width: 100%;
+
+  /* Padding based on size */
+  padding: ${(p) => cardPadding[p.$size]}px;
+
+  /* Border styling */
+  border: ${(p) => (p.$hasBorder ? `1px solid ${alphas.black10}` : 'none')};
+
+  /* Shadow styling */
+  box-shadow: ${(p) => (p.$hasShadow ? `0 2px 8px ${alphas.black10}` : 'none')};
+
+  /* Clickable styling */
+  cursor: ${(p) => (p.$isClickable ? 'pointer' : 'default')};
+  transition: ${(p) => (p.$isClickable ? 'box-shadow 0.2s ease, transform 0.2s ease' : 'none')};
+
+  &:hover {
+    ${(p) =>
+      p.$isClickable
+        ? `
+      box-shadow: 0 4px 12px ${alphas.black20};
+      transform: translateY(-1px);
+    `
+        : ''}
+  }
+
+  &:active {
+    ${(p) =>
+      p.$isClickable
+        ? `
+      transform: translateY(0);
+      box-shadow: 0 2px 8px ${alphas.black10};
+    `
+        : ''}
+  }
+`;
+
+const RelativeContainer = styled.div`
+  position: relative;
+  width: 100%;
+  height: 100%;
+`;
+
+const ActionButtonContainer = styled.div<{zIndex?: number; $showOnHover?: boolean}>`
+  position: absolute;
+  top: 8px;
+  right: 8px;
+  z-index: ${(p) => p.zIndex || 10};
+  display: flex;
+  align-items: center;
+  gap: 4px;
+  opacity: ${(p) => (p.$showOnHover ? 0 : 1)};
+  transition: opacity 0.2s ease;
+
+  /* Show on parent hover (only if showOnHover is true) */
+  ${(p) =>
+    p.$showOnHover &&
+    `${RelativeContainer}:hover & {
+    opacity: 1;
+  }`}
+`;
+
+/*
+ * Component.
+ */
+
+const CardComponent: FC<CardProps> = ({
+  children,
+  size = VisualSizesEnum.MEDIUM,
+  hasShadow = true,
+  hasBorder = false,
+  className,
+  isClickable = false,
+  onClick,
+  actions = [],
+  showActionsOnHover = false,
+  groupActions = false
+}) => {
+  const handleClick = () => {
+    if (isClickable && onClick) onClick();
+  };
+
+  const handleActionClick = (action: CardAction) => {
+    action.onClick();
+  };
+
+  // If no actions, render the card normally
+  if (actions.length === 0)
```

Comment:
Similar to https://github.com/frontapp/front-ui-kit/pull/286#discussion_r2446116150, I think we could also return `null` in the `renderActions` method